html-forms/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Forms</title>
+</head>
+<body>
+  <form action=>
+    <label for="email" >Email</label>
+    <input type="email" id="email" name="email"/>
+
+
+    <label>Why is coding cool ?</label>
+    <select name="options">
+    <option value="Interesting"> Because it's interesting</option>
+    <option value="fun">Because it's fun</option>
+    <option value="Powerful">Because it's powerful</option>
+    </select>
+
+
+    <label>Favourite language (choose one)</label>
+    <input type="radio" name="language" value="HTML" check="checked"/> HTML
+    <input type="radio" name="language" value="css" check="checked" /> CSS
+    <input type="radio" name="language" value="js" check="checked" /> Javascript
+
+    <label>Would you like to sign up for more information?</label>
+    <input type="checkbox" name="sign me up" value="sign me" checked="checked"/> Sign me up
+
+    <label>Comments</label>
```

Comment:
Every `label` element should have a `for` attribute corresponding to an `id` on the input it is labelling.